seg_utils/ui/main_window.py

```diff
@@ -214,6 +233,38 @@ def import_file(self, existing_patients: list):
                     self.sAddFile.emit(filepath, patient)
                     self.sRequestUpdate.emit(self.img_idx)
 
+    def new_project(self):
+        """executes a dialog prompting the user to enter information about the new project"""
+        if self.check_for_changes():
+            dlg = ProjectHandlerDialog()
+            dlg.exec()
+            if dlg.project_path:
+                database_path = dlg.project_path + Structure.DATABASE_DEFAULT_NAME
+                self.sCreateNewProject.emit(database_path, dlg.files)
+                self.menubar.enable_tools()
+
+    def open_project(self):
+        """executes a dialog prompting the user to select a database"""
+        if self.check_for_changes():
+            """database, _ = QFileDialog.getOpenFileName(self,
+                                                      caption="Select Database",
+                                                      directory=str(Path.home()),
+                                                      filter="Database (*.db)",
+                                                      options=QFileDialog.DontUseNativeDialog)"""
+            database = '/Users/jakob/AnnotationProjects/project14/database.db'
```

Comment:
Please remove this so it works as it should before we merge it.

Reply:
done